src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
@sophistifunk if you're curious about where I was originally going with the "composed component" approach you can view commit d1bea54 which I later reverted.